config/initializers/node-orm2-connect.js

```diff
@@ -6,11 +6,16 @@ export default {
   before: 'define-orm-models',
   async initialize(application) {
     let container = application.container;
-    let adapter = container.lookup('orm-adapter:node-orm2');
-    let config = application.config.database;
+    let config = application.config;
+
+    if (!config.database || !config.database.orm2) {
+      // Config is not specified
+      return;
+    }
 
     try {
-      adapter.db = await fromNode((cb) => orm.connect(config.url, cb));
+      let connection = await fromNode((cb) => orm.connect(config.database.orm2, cb));
+      container.register('database:orm2', connection, { singleton: true });
```

Comment:
Two things:

1. I don't _love_ storing a connection object in the container. I get why it's necessary, and that's fine. But I'd love to brainstorm a better approach here. The db connection is something I could see sharing across containers (potentially). It's typically managed by the ORM library, and since ava runs tests in parallel, I've noticed that it's very easy to start to hit connection limits (i.e. `Error: too many clients`) because each individual test creates a db connection. As long as the ORM doesn't store any state on the connection that could be shared across containers as well.

2. If we _do_ keep the connection object in the container, I'd like to settle on a naming convention for where we put it - I'm okay with `database` as the type, or `connection` or something like that.